concepts/basics/introduction.md

```diff
@@ -53,31 +60,52 @@ int always_fortyseven() {
 }
 ```
 
-Or `total_fee`, a function with three parameters.
+Here is `vip_fee`, which has one parameter:
```

Comment:
are we calling them parameters or arguments?

Reply:
When a function has parameters, you can pass it arguments.